Makefile

```diff
@@ -146,6 +166,7 @@ revendor:
 	@GO111MODULE=on cd third_party/kube-scheduler/v19; go mod tidy
 	@GO111MODULE=on cd third_party/kube-scheduler/v20; go mod tidy
 	@GO111MODULE=on cd third_party/kube-scheduler; go mod tidy
+	@cd vendor/k8s.io/code-generator; go mod vendor
```

Comment:
Hm, why is this needed?

Reply:
On my NixOS I could not get it building because when the code-generator was build, it never had it's dependencies available and was failing with typical cryptic go error messages. It complained that it needed vendoring and this fixed it.
go does not walk, limps ....

Reply:
What version of go are you using?
Is the gardener in your ${GOPATH}?

Reply:
go version go1.15.5 linux/amd64

Reply:
Shouldn't it be `~/go/src/...`? what's the value of your `$GOPATH`?

Reply:
@rfranzke ups, it is of course ~/go/src/github.com/gardener/gardener